assets/dev/js/editor/components/ai/pages/connect/index.js

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useRef } from 'react';
+import { Stack, Button, Typography, Link } from '@elementor/ui';
+import { AIIcon } from '@elementor/icons';
+
+const Connect = ( { connectUrl, onSuccess } ) => {
+	const approveButtonRef = useRef();
+
+	useEffect( () => {
+		jQuery( approveButtonRef.current ).elementorConnect( {
+			success: ( _, data ) => {
+				elementorCommon.config.library_connect.is_connected = true;
+				elementorCommon.config.library_connect.current_access_level = data.accessLevel;
+
+				onSuccess();
+			},
+			// eslint-disable-next-line no-console
+			error: () => console.error( 'Error connect' ),
+		} );
+	}, [] );
+
+	return (
+		<Stack alignItems="center" gap={ 5 }>
+			<AIIcon sx={ { color: 'white', fontSize: '60px', mb: 3 } } />
+
+			<Typography variant="h4">{ __( 'Step into the future with Elementor AI', 'elementor' ) }</Typography>
+
+			<Typography variant="body2">{ __( 'Create smarter with AI text and code generators built right into the editor.', 'elementor' ) }</Typography>
+
+			<Typography variant="caption" sx={ { maxWidth: 520, textAlign: 'center' } }>
+				{ __( 'By clicking "Connect", I approve the ', 'elementor' ) }
+				<Link href="https://go.elementor.com/ai-terms/" target="_blank" color="info.main">{ __( 'Terms of Service', 'elementor' ) }</Link>
+				{ ' & ' }
+				<Link href="https://go.elementor.com/ai-privacy-policy/" target="_blank" color="info.main">{ __( 'Privacy Policy', 'elementor' ) }</Link>
+				{ __( ' of the Elementor AI service.', 'elementor' ) }
```

Comment:
not sure about the translation of all of it, I think it should be a single string somehow